docs/about/governance.rst

```diff
@@ -1,54 +1,43 @@
+Once the user needs for a new feature in BODS have been described in a feature development ticket, proposals can be made about how to implement the feature. Currently Open Ownership and Open Data Services use GitHub Discussions to get feedback about `implementation proposals <https://github.com/openownership/data-standard/discussions/categories/feature-implementation>`_.
 
-**Expert**
+If broad consensus about whether or not to proceed with implementation as laid out in the proposal can't be reached, the decision should be postponed.
 
-We will:
+Release tracking
+----------------
 
-* bring our expertise to the discussion as individuals;
-* use our expertise to synthesise the views of others in constructive and forward-thinking proposals;
-* use good judgement to respect privacy and confidentiality.
+For each new version of BODS, Open Ownership and Open Data Services will create a release tracker as a `project <https://github.com/orgs/openownership/projects>`_ of the `Open Ownership GitHub team <https://github.com/openownership>`_ to capture the features, bug fixes, changes and updates being made in the new version. Here is the `release tracker for BODS version 0.4 <https://github.com/orgs/openownership/projects/4/views/1>`_ as an example.
 
-**Collaborative**
+Data Standard Working Group
+---------------------------
 
-We will
-
-* support each other in discussion, in decisions, and in delivery;
-* operate on the basis of consensus decision making; 
-* constructively hold each other to account on our commitments;
-* ensure all voices are heard and considered carefully.
-
-Membership, term and role
--------------------------
-
-Membership of the working group is a voluntary responsibility. Members should be prepared to join quarterly virtual group calls and respond to issues via GitHub or email.
+The Data Standard Working Group brings together in-country, technical and data experts to help shape the development of BODS as well as driving adoption of the standard internationally.
 
+This working group is open to all and includes stakeholders from a range of backgrounds with an interest and expertise in data relating to corporate control or beneficial ownership, and to the use of such data in the public interest.
 
+Membership of the working group is voluntary. Anyone can apply to join the group by `filling out this form <https://docs.google.com/forms/d/e/1FAIpQLSdRSmSUxyyv2t1k3vWXZ_3EhTW_f603MeGxgyjKnbNNE9vvbQ/viewform>`_ after consulting the `terms of reference <https://docs.google.com/document/d/1kNxojFM0BLZvUYlFWfayQ88CG7ffC-ypjMtrZxDDKmU/edit?usp=sharing>`_. Virtual group meetings are held quarterly and communication is coordinated through a `Google group <https://standard.openownership.org/en/0.3.0/about/governance.html>`_. Members are encouraged to respond to issues via `GitHub <https://github.com/openownership/data-standard/issues>`_ or email.
```

Comment:
`Google group <https://standard.openownership.org/en/0.3.0/about/governance.html>` does this link need to be updated or should it be pointing back at the governance page?

Reply:
I just removed the link. I didn't correct it to point to the Google Group since that would encourage people to sign up directly, and we'd like them to use the form.